src/coreclr/vm/peimage.cpp

```diff
@@ -75,19 +75,7 @@ CHECK PEImage::CheckLayoutFormat(PEDecoder *pe)
     }
     CONTRACT_CHECK_END;
 
-    // If we are in a compilation domain, we will allow
-    // non-IL only files to be treated as IL only
-
-    // <TODO>@todo: this is not really the right model here.  This is a per-app domain
-    // choice, but an image created this way would become available globally.
-    // (Also, this call prevents us from moving peimage into utilcode.)</TODO>
```

Comment:
Now that this comment has been resolved, do we want to look at moving PEImage into utilcode, or is that irrelevant nowadays?

Reply:
Filed https://github.com/dotnet/runtime/issues/57793 for looking into moving it.